paper.md

```diff
@@ -76,4 +76,41 @@ Figures can be included like this: ![Fidgit deposited in figshare.](figshare_art
 
 # Main Body
 
+## learning objectives and content
+The overarching objective of the course is to teach quantitative research
+methods that are reproducible and collaborative. Each lecture is independent and
+have specific learning objectives. First lecture starts with giving students an
+overview of the capabilities of R such as more efficient quantitative analysis
+of data compared to manual handling of spreadsheets, programming in R, and
+integrating analyses with generating reports using R Notebook (markdown syntax).
+Second lecture introduces fundamental elements in programming with R such as
+vectors, data frames, basic operations and use of functions. The third lecture
+intorduces the concept of exploratory analysis of datasets with a focus on using
+dplyr package. The objective for lecture four is to learn the advantage of using
+summary statistics and data visualization using tidyverse packages. Lecture five
+focuses on approaches for cleaning up and preprocessing of raw data. Lecture
+seven introduces basic descriptive and inferential statistical techniques such
+as linear models. Lecture eight focuses on linear mixed effects models to
+analyze the fixed and random effects in nested exploratory data. Lecture nine
+introduces randomization tests and teaches simulating datasets and applying such
+tests on simulated and read data. Lecture ten's objective is to learn
+multivariate statistics with a focus on principle component analysis and
+multivariate analysis of variance. Lecture eleven introduces model selection
+approach, its benefits over traditional inferential statistics, and model
+selection criteria such as Akaike’s Information Criterion. Lecture twelve is to
+learn population models and using of differential equations such to analyze
+system dynamics. Lecture thirteen focuses on time series data and fitting
+numerical models to such datasets. Lecture fourteen introduces the necessity to
+acquire a scientific approach, the steps involved and integrating such approach
```

Comment:
`Lecture fourteen introduces the scientific method and integrating scientific approaches with group dynamics`